packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -56,6 +57,13 @@ export function TransactionsTableRow({
     return () => clearInterval(interval)
   }, [tx])
 
+  useEffect(() => {
+    const secondsPassed = dayjs().diff(dayjs(tx.createdAt), 'second')
+    if (secondsPassed <= 20) {
+      setShouldHighlightTx(true)
+    }
+  }, [tx])
```

Comment:
so the way this is set up -
if i have already cancelled the blink (hover / click) on the row
but immediately do another tx
the "cancelled highlight" row will also blink

Reply:
I refactored this, now we add the class in the parent component